src/plugins/intel_cpu/src/nodes/interpolate.cpp

```diff
@@ -159,6 +159,10 @@ struct jit_uni_interpolate_kernel_f32 : public jit_uni_interpolate_kernel, publi
             case InterpolateMode::bilinear_pillow:
             case InterpolateMode::bicubic_pillow: {
                 switch (jcp_.layout) {
+                    case InterpolateLayoutType::planar: {
```

Comment:
If I understand it right, we've already supported nspc layout for interpolate pillow mode, now we have the case with ncsp layout that can be perfectly reinterpreted as nspc layout. Then maybe JIT kernel doesn't have to be aware of such reinterpretation. Just reset layout to be nspc where you reset shape dims. So JIT kernel will treat it like a normal nspc case, which will make it easier for further JIT kernel maintaining. Or did I miss anything?

Reply:
I think this is a good suggestion. What's more, the mapping could be even totally out of executor scope, just align in prepareParams(). All is done in [548e0ae](https://github.com/openvinotoolkit/openvino/pull/17079/commits/548e0ae817ac1a64f14b89d324806b9e8eb9b936). Thank you!